en/console-and-shells/completion-shell.rst

```diff
@@ -94,20 +106,50 @@ The following bash example comes from the original author::
             opts=$(${cake} Completion subcommands $prev)
             COMPREPLY=( $(compgen -W "${opts}" -- ${cur}) )
             if [[ $COMPREPLY = "" ]] ; then
-                COMPREPLY=( $(compgen -df -- ${cur}) )
+                _filedir
```

Comment:
With my MacOS' internal knowlegde being really bad at best, I could not find a way to check this.
Apparently, it can be installed using homebrew, but I'm unsure at this point.

Maybe someone with a better knowledge of MacOS can write a setup guide to enable autocompletion on Mac ?